ft/tests/mock_command.py

```diff
@@ -1,34 +1,37 @@
-from nose.tools import assert_equals
-
 from ft.internal import ftformat
 from ft.functions import function_list
 
 
-def mock_command(cmd):
+def mock(cmd, input=None, **params):
```

Comment:
These are optional but I'd say it makes sense to pass these parameters here.
Also makes the tests less imperative.